autopush/router/fcm.py

```diff
@@ -195,6 +196,12 @@ def _route(self, notification, router_data):
         except pyfcm.errors.AuthenticationError as e:
             self.log.error("Authentication Error: %s" % e)
             raise RouterException("Server error", status_code=500)
+        except ConnectionError as e:
```

Comment:
Yeah, I was debating that. There's a lot of connection related exceptions. ConnectTimeout is extended from ConnectionError, so it should be captured here, along with ProxyError and SSLError. 
The other root exception is RequestError, which is tempting to include, but that captures things that we absolutely want raised as errors.